jina/executors/indexers/dbms/keyvalue.py

```diff
@@ -0,0 +1,94 @@
+import pickle
+from typing import List, Optional
+
+from jina.drivers.dump import DumpPersistor, DumpTypes
```

Comment:
Executors should not import from `Drivers` I think.

Is better to have a special module in executors for `dump`.

Reply:
Done, check again if you can